x/staking/keeper/delegation.go

```diff
@@ -786,7 +786,7 @@ func (k Keeper) CompleteUnbonding(ctx sdk.Context, delAddr sdk.AccAddress, valAd
 	// loop through all the entries and try to complete unbonding mature entries
 	for i := 0; i < len(ubd.Entries); i++ {
 		entry := ubd.Entries[i]
-		if entry.IsMature(ctxTime) && !entry.UnbondingOnHold {
+		if entry.IsMature(ctxTime) && !entry.OnHold() {
```

Comment:
Minor nitpick- why not call it `UnbondingOnHold()`?

Reply:
My idea was to keep it consistent with `IsMature` (it is not `UnbondingIsMature`). The Unbonding Delegation Entry is matured or on hold. But I don't have a strong position about it. @jtremback Let me know if I should change to `UnbondingOnHold()`.